com.unity.formats.fbx/Editor/FbxExporter.cs

```diff
@@ -1745,23 +1752,9 @@ internal static HashSet<float> GetKeyTimes(AnimationCurve[] animCurves)
         /// NOTE : This is a work in progress (WIP). We only export the key time and value on
         /// a Cubic curve using the default tangents.
         /// </summary>
-        internal void ExportAnimationKeys (AnimationCurve uniAnimCurve, FbxAnimCurve fbxAnimCurve, 
-            UnityToMayaConvertSceneHelper convertSceneHelper)
+        internal static void ExportAnimationKeys (AnimationCurve uniAnimCurve, FbxAnimCurve fbxAnimCurve, 
+            UnityToMayaConvertSceneHelper convertSceneHelper, string uniPropertyName)
```

Comment:
is it me or the uniPopertyName parameter is not being used anywhere?

Reply:
you're right I don't use it anymore, good catch